.circleci/config.yml

```diff
@@ -41,37 +41,38 @@ jobs:
     # This is based on your 1.0 configuration file or project settings
     - run:
         working_directory: ~/orbitdb/orbit-db
-        command: nvm install 8.2.0 && nvm alias default 8.2.0
+        command: nvm install 10.13.0 && nvm alias default 10.13.0
```

Comment:
Since we're now going to support Node.js 10, we should modify the relevant field in package.json and README (8->10)

Reply:
Changed in package.json now and README will pick it up from npm